### ISSUE TITLE
Plugin preset files are not allowed to export objects only functions

### ISSUE THREAD

**Issue by raptorf1** [CraftAcademy/CraftOverflow#38]
## Problem
Continued by Felix's and Zane's post [here](https://github.com/CraftAcademy/CraftOverflow/pulls).

### Background
Tried today to start with the Paper-Rock-Scissors assignment of ours. Decided to use cucumber and the TDD methods we used during the address book challenge.

### Story
For those of you that decide to follow the same approach and in order to not loose 2 hours of your time either :( please read below.

After following to the letter our material and doing what Zane and Felix describe above, I got the error of the attached picture.
After searching and trying various things, what solved the problem was [this](https://babeljs.io/docs/en/env) documentation.

###Screenshots

![3](https://user-images.githubusercontent.com/48210172/56389500-04315080-622a-11e9-8f95-a93a1b9031e1.jpg)

### How did you try to solve the problem?
After you do the steps in the material and go through what Zane and Felix describe, instead of
```
$ yarn add webpack webpack-cli babel-loader babel-preset-es2015 babel-polyfill babel-register --dev
```
you run
```
$ yarn add webpack webpack-cli babel-loader babel-preset-env babel-polyfill babel-register --dev
```
Upon successful completion, you go to your webpack.config.js file and change the line `presets: ["es2015"]` to `presets: ["env"]`.
If you run `yarn run build` now it is working.

**Comment by raptorf1** [comment]
solved

**Comment by tochman** [comment]
Really good job @raptorf1

**Comment by raptorf1** [comment]
And the story continues!!!!!

![Screenshot from 2019-04-19 12-29-08](https://user-images.githubusercontent.com/48210172/56420586-ddf9c800-629e-11e9-8100-04b1104587eb.png)

Today that the app.js got some code in its lines, I was against the error above!

Solved by first deleting `node_modules` and `yarn.lock` and then installing `yarn add babel-loader @babel/core @babel/preset-env webpack`. Then you go to the package json.file and make this change.
```
options: {
              presets: ["@babel/preset-env"]
            }
```

Then `yarn build` throws no error.

The problem continues though cause even after the successful build message my function did not work. If I put the same function in the HTML file all is fine.

And just now I tried to write the function in ES6 mode and all is good. So write your functions in ES6 (arrow functions).
